addons/payment_razorpay/views/payment_provider_views.xml

```diff
@@ -22,6 +22,14 @@
                            password="True"/>
                 </group>
             </group>
+            <field name="allow_tokenization" position='after'>
+                <div colspan="2" invisible="code != 'razorpay'">
+                    <a href="https://razorpay.com/docs/api/payments/recurring-payments/" class="btn btn-link" role="button" target="new">
```

Comment:
We should maybe point to an anchor in our own documentation to control and rephrase the content, and ensure the link will not break.

cc @dalaOdoo ?